html/includes/graphs/device/agent.inc.php

```diff
@@ -19,9 +18,11 @@
 $colour_area_max = "FFEE99";
 
 $graph_max = 1;
+$multiplier = 1000;
+$multiplier_action = "/";
 
-$unit_text = "msec";
+$unit_text = "Seconds";
```

Comment:
Why did the unit-text changed from Milliseconds to Seconds ? (Was it wrong all the time?)